docs/pages/build-reference/variables.md

```diff
@@ -61,9 +63,92 @@ See the [eas.json reference](/build/eas-json.md) for more information.
 
 ## Using secrets in environment variables
 
-🚧 We are currently working on a secrets API that will allow developers to store generic encrypted secrets and selectively expose them to build jobs. This feature will be available before EAS Build graduates from preview.
+To provide your build jobs with access to values that are too sensitive to include in your source code and git repository, you can use "Secrets".
+
+These secrets are encrypted at rest and in transit, and are only decrypted in a secure environment by EAS servers.
+
+It is possible to create up to 100 account-wide secrets for each Expo account and 100 app-specific secrets for each app (you probably won't need that many). Account-wide, or "global" secrets will be exposed to every build environment across all of your apps. App-specific secrets apply to the app they're defined on, and override global secrets with the same name.
+
+You can manage secrets through the Expo website and EAS CLI.
+
+### Secrets on the Expo website
+
+To create secrets to use across all apps in an account, navigate to the "Secrets" tab under the [account settings](https://expo.io/settings/secrets):
+
+<ImageSpotlight alt="Global secrets location" src="/static/images/eas-build/environment-secrets/secrets-account-nav.png" />
+
+To create secrets to in a specific project, navigate to the "Secrets" tab under the project dashboard:
+
+<ImageSpotlight alt="Project secrets location" src="/static/images/eas-build/environment-secrets/secrets-project-nav.png" />
+
+If you haven't published your project yet and it isn't visible on the website, create one on the project list page.
+
+<ImageSpotlight alt="Create project button location" src="/static/images/eas-build/environment-secrets/project-creation-navigation.png" />
+
+<ImageSpotlight alt="Create project UI" src="/static/images/eas-build/environment-secrets/project-creation-web.png" />
+
+### Adding secrets with the website
+
+When setting up secrets for a new account or app, you'll be met with this UI:
+
+<ImageSpotlight alt="Empty secrets UI" src="/static/images/eas-build/environment-secrets/secrets-empty.png" />
+
+Click the "Create" button in the top-right of the table to create a new secret.
+
+A secret needs a name and a value. The name can only contain alphanumeric characters and underscores:
+
+<ImageSpotlight alt="Secret creation UI filled" src="/static/images/eas-build/environment-secrets/secrets-create-filled.png" />
+
+<ImageSpotlight alt="Secret UI with stored secret" src="/static/images/eas-build/environment-secrets/secrets-populated.png" />
+
+### Adding secrets with EAS CLI
+
+To create a new secret, run `eas secrets:create`
+
+```
+> eas secrets:create project SECRET_NAME secretvalue
```

Comment:
is `project` a flag here, or should you put the actual slug of your project?

Reply:
it's a flag